app/src/main/java/org/fossasia/openevent/general/auth/EditProfileFragment.kt

```diff
@@ -55,14 +55,16 @@ class EditProfileFragment : Fragment() {
                 val imageUrl = it.avatarUrl.nullToEmpty()
                 rootView.firstName.setText(userFirstName)
                 rootView.lastName.setText(userLastName)
-                context?.let { ctx ->
-                    val drawable = AppCompatResources.getDrawable(ctx, R.drawable.ic_account_circle_grey_24dp)
-                    drawable?.let { icon ->
-                        Picasso.get()
-                            .load(imageUrl)
-                            .placeholder(icon)
-                            .transform(CircleTransform())
-                            .into(rootView.profilePhoto)
+                if(!imageUrl.isEmpty()) { // picasso requires the imageUrl to be non empty
```

Comment:
Space after if

Reply:
@iamareebjamal Sorry, missed that. I have removed it

Reply:
@iamareebjamal is it correctly done now ?